src/test/kotlin/calculator/ExpressionKtTest.kt

```diff
@@ -0,0 +1,18 @@
+package calculator
+
+import io.kotest.assertions.throwables.shouldThrow
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.NullSource
+import org.junit.jupiter.params.provider.ValueSource
+
+class ExpressionKtTest {
+
+    @ParameterizedTest
+    @NullSource
+    @ValueSource(strings = ["", " ", "  ", "\n"])
```

Comment:
Empty의 케이스도 애너테이션으로 제공하는 `@NullAndEmptySource`를 활용해볼 수 있을 것 같아요. 😃
```suggestion
    @NullAndEmptySource
    @ValueSource(strings = [" ", "  ", "\n"])
```